### PR TITLE
this fixes "flutter build apk" in the google_sign_in plugin

### PR DESCRIPTION
**Pull request by andretietz**
Makes sure "flutter build apk" is building the apk.

https://github.com/flutter/flutter/issues/13966

### CODE REVIEW

File: packages/google_sign_in/example/android/app/build.gradle [hunk 1]
```diff
@@ -33,6 +33,9 @@ android {
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
         }
+        profile {
+            matchingFallbacks = ['debug', 'release']
```
Comment (mravn-google) [inline packages/google_sign_in/example/android/app/build.gradle]: This is not necessary with Flutter master where it is part of the `flutter.gradle` script in Flutter tools.
We should either not do that here (just wait for alpha to roll), or do it on all plugins.

Comment (mravn-google) [reply packages/google_sign_in/example/android/app/build.gradle]: A fix of alpha roll blocker https://github.com/flutter/flutter/issues/13742 is ready, so an alpha roll should be just around the corner.

**Comment by Hixie** [comment]
We will be rolling alpha on Tuesday (and renaming it dev! Exciting yet ultimately meaningless changes afoot! :smile:). We try to avoid rolling just before weekends or holidays where many of the Flutter team are away, and in the US Monday is a holiday.

**Comment by Hixie** [comment]
(In the meantime feel free to run `flutter channel master` to switch to the current build, which is more or less what we'll roll to dev.)

Should this PR be closed in the meantime?

**Comment by Hixie** [comment]
We've finally rolled dev. Sorry it took so long.

**Pull request closed without merging**